rasa/model.py

```diff
@@ -187,9 +187,12 @@ def unpack_model(
         working_directory = tempfile.mkdtemp()
 
     # All files are in a subdirectory.
-    with tarfile.open(model_file, mode="r:gz") as tar:
-        tar.extractall(working_directory)
-    logger.debug(f"Extracted model to '{working_directory}'.")
+    try:
+        with tarfile.open(model_file, mode="r:gz") as tar:
+            tar.extractall(working_directory)
+            logger.debug(f"Extracted model to '{working_directory}'.")
+    except Exception as e:
+        logger.error(f"Failed to extract model at {model_file}. Error: {e}")
```

Comment:
should we really go on like nothing happened? Isn't it better to fail with the exception?

Reply:
Ah good point. I guess i wanted to improve the message (didn't tell you where it was trying to extract from), but skipped the raising in the process. I guess i should do sth like 
```
except Exception as e:
    raise type(e)(e.message + "\n Failed to extract model at {model_file}.")
```

Reply:
what's the current message? Should the zipping library have a proper error message?

Reply:
Nvm found sth in agent:
```
    except Exception as e:
        logger.error(f"Could not load model due to {e}.")
        raise
```

Reply:
At this point I don't remember, but I added it bc i ran into the issue and it didn't give me the path 😄would have to try to make it break again haha

Reply:
shouldn't it be `raise e`?

Reply:
looks like you `raise e` if you add it to the output itself instead of a log:
```
        e.message += (
            f". Failed to validate training data, make sure your data "
            f"is valid. For more information about the format visit "
            f"{DOCS_BASE_URL}/nlu/training-data-format/"
        )
        raise e
```
do you have a preference of logger versus appending the exception? looks like we do both in our code

Reply:
or rather because i already included `e` in my error msg

Reply:
just read that `raise` is enough (so no `raise e` needed). 

> do you have a preference of logger versus

it doesn't matter - it'll show up anyway. Just go for one of the options